.github/ISSUE_TEMPLATE/propose_new_rule.yml

```diff
@@ -0,0 +1,88 @@
+name: Propose a new rule
+description: Propose a new rule for the eslint-plugin-testing-library.
+labels: [new rule]
+body:
+  - type: input
+    id: name_for_new_rule
+    attributes:
+      label: Name for new rule
+      description: Suggest a name for the new rule.
```

Comment:
Can we add links here? Would be amazing if we could attach the [Rule naming conventions from CONTRIBUTING](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/CONTRIBUTING.md#rule-naming-conventions)

Reply:
```suggestion
      description: Suggest a name for the new rule that follows the [rule naming conventions](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/CONTRIBUTING.md#rule-naming-conventions).
```